code/modules/modular_computers/ntos/ntos.dm

```diff
@@ -63,10 +64,10 @@
 		if(hard_drive)
 			if(prob(10))
 				hard_drive.visible_message("<span class='warning'>[src] emits some ominous clicks.</span>")
-				hard_drive.take_damage(0.5 * hard_drive.health)
+				hard_drive.take_damage(20)
```

Comment:
why hard numbers here?
health of components hadn't been really tested in live env so might end up being tweaked, I think it's better to leave stuff like this with multipliers